pygfx/controllers/_base.py

```diff
@@ -1,6 +1,7 @@
 from typing import Tuple, Union
 
-from ..linalg import Vector3
+import pylinalg.func as la
```

Comment:
You don't need the `.func` part :)

Reply:
Haha, thanks.

Related, are we going for `as la` or `as pla`? I started with the former, but the pylinalg tests use the latter.

Reply:
I guess `la` is preferable, just like it's `np` and not `npy`.